kong.conf.default

```diff
@@ -185,6 +185,27 @@
                                   # `text/html`, `application/json`, and
                                   # `application/xml`.
 
+#client_max_body_size = 0        # Defines the maximum request body size allowed
```

Comment:
Using the Nginx default (`1m`) would be a breaking change, as we already define this as `0` in our template, hence the use of 0 here- my initial thought was to set it at `1m`, but this breaks tests and a number of expected behaviors. Would be happy to set this to `1m` in a separate change targetting 0.11 in a separate PR.